src/main/kotlin/view/OutputView.kt

```diff
@@ -0,0 +1,20 @@
+package view
+
+import domain.player.Player
+import domain.player.Players
+
+object OutputView {
+    fun firstDrawTo(players: Players) {
+        println("${players}에게 2장의 카드를 나누었습니다.")
+    }
+
+    fun printHandsOf(player: Player) {
+        println("${player.name} 카드 : ${player.handsStatus()}")
+    }
+
+    fun gameResult(players: Players) {
+        repeat(players.size()){
+            println("${players[it].name} 카드 : ${players[it].handsStatus()} - 결과 : ${players[it].score()}")
+        }
+    }
```

Comment:
I/O는 성능 병목에 영향을 미치는 부분인데요.
반드시 반복될때마다 출력을 해줘야하는 것이 아니라면 출력할 메세지를 모아서 한번만 출력하는건 어떨까요?
kotlin에서 with을 이용하면 아래와 같이 변경할 수 있습니다.

https://kotlinlang.org/docs/scope-functions.html#with

```suggestion
    fun gameResult(players: Players) =with(StringBuilder()) {
        repeat(players.size()){
            appendLine("${players[it].name} 카드 : ${players[it].handsStatus()} - 결과 : ${players[it].score()}")
        }
        print(this)
    }
```